sdk/textanalytics/ai-text-analytics/src/index.ts

```diff
@@ -139,5 +141,6 @@ export {
   PiiCategory,
   Association,
   Certainty,
-  Conditionality
+  Conditionality,
+  RelationType
```

Comment:
`HealthcareEntityRelationType`, in the future, we add other relation types, then it won't works

Reply:
Done.